Weather/Weather/Scenes/Search/SearchLocationViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SearchLocationViewController.swift
+//  Weather
+//
+//  Created by minh duc on 2/19/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import UIKit
+
+class SearchLocationViewController: BaseTableViewController {
```

Comment:
```suggestion
final class SearchLocationViewController: BaseTableViewController {
```
thêm final để ngăn chặn override